scrape/scrape.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+from exceptions import NeedYearArgumentOnlyException, NonNumericYearArgumentException
+from typing import Dict, Union, List, Tuple
+from bs4 import BeautifulSoup
+from calendar_dict import INT_TO_MONTH, INT_TO_WEEKDAY_DICT
+
+import requests
+import json
+import sys
+import os
+import datetime
+import time
+import bs4
+
+# Example: python scrape.py 2080 fetches data for 2080B.S and saves them in database/2080.json
+
+def fetch_and_parse(year : int) -> Dict[str, Dict]:
+
```

Comment:
Use a formatter like `yapf` to enforce consistent formatting. Also add `yapf` configuration file in the root.